apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue

```diff
@@ -75,12 +76,13 @@ const fields = computed<DefinitionField[]>(() => [
     { name: 'last_collected_at', label: i18n.t('INVENTORY.COLLECTOR.DETAIL.LAST_COLLECTED') },
     { name: 'created_at', label: i18n.t('INVENTORY.COLLECTOR.DETAIL.CREATED') },
 ]);
-const collector = ref<null|CollectorModel>(null);
-const loading = ref(true);
 
-const plugins = computed<PluginReferenceMap>(() => store.getters['reference/pluginItems']);
-const pluginName = computed<string>(() => (collector.value ? plugins.value[collector.value.plugin_info.plugin_id]?.label ?? '' : ''));
-const pluginIcon = computed<string>(() => (collector.value ? plugins.value[collector.value.plugin_info.plugin_id]?.icon ?? '' : ''));
+const state = reactive({
+    collector: null as null|CollectorModel,
+    loading: true,
+    pluginName: computed<string>(() => (state.collector ? plugins.value[state.collector.plugin_info.plugin_id]?.label ?? '' : '')),
+    pluginIcon: computed<string>(() => (state.collector ? plugins.value[state.collector.plugin_info.plugin_id]?.icon ?? '' : '')),
+});
```

Comment:
![after](https://github.com/cloudforet-io/console/assets/18563857/7af0d00a-792d-4d45-96c5-edf8b6c0396a)

Reply:
I like it.

Reply:
![image](https://github.com/cloudforet-io/console/assets/83635051/ccbce6e9-e1db-4776-a1c2-f9af926e9fee)